PetAdoptions/cdk/pet_stack/lib/services.ts

```diff
@@ -42,6 +42,10 @@ export class Services extends Stack {
         const defaultSecondaryCIDR = this.node.tryGetContext('vpc_cidr_secondary') || "10.2.0.0/16";
         const fisLambdaTagName = this.node.tryGetContext('fisLambdaTagName') || 'FISExperimentReady';
         const fisLambdaTagValue = this.node.tryGetContext('fisLambdaTagValue') || 'Yes';
+        const fisLambdaExtensionArn = this.node.tryGetContext('fisLambdaExtensionArn')||'arn:aws:lambda:us-east-1:211125607513:layer:aws-fis-extension-x86_64:9';
```

Comment:
Add this to the context file, if you are trying to read from it.
That is /PetAdoptions/cdk/pet_stack/cdk.json